Indonesian/botol-merah/minum.md

```diff
@@ -0,0 +1,9 @@
+anda mengetahui bahwa diri anda akan menemukan cinta sejati anda 
+dan istri anda akan mengandung seorang anak. anda baru tahu bahwa istri anda terkena kanker rahim.
+namun saat melahirkan istri anda akan meninggal dunia dan apabila melahirkan anak yang anda akan menemukan obat kanker.
+namun apabila tidak melahirkan istri anda lima tahun kemudan akan meninggal dunia.
+Apa yang anda lakukan.
+
+apabila anda ingin menggugurkan kandungan istri anda [klik disini](../gugur/gugurkan-kandungan.md)
+apabila anda ingin membiarkan anak anda lahir [klik disini](../lahir/biarkan-melahirkan.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.